src/test/java/no/sikt/nva/scrapers/embargo/OnlineEmbargoCheckerImplTest.java

```diff
@@ -0,0 +1,96 @@
+package no.sikt.nva.scrapers.embargo;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import java.net.http.HttpClient;
+import java.net.http.HttpClient.Redirect;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+@Disabled
```

Comment:
Using @Disabled annotation, it seems like there does not exists something like `@Online` annotation

Reply:
Using RemoteTest annotation